### PR TITLE
zh-trans: update CronJob

### PR DESCRIPTION
**Pull request by mysunshine92**
zh-trans: update CronJob

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -157,6 +157,26 @@ the Job in turn is responsible for the management of the Pods it represents.
...
+
+```
+--feature-gates="CronJobControllerV2=true"
+```
+
+
+-->
+## 新控制器
+
+自 Kubernetes 1.20 开始以 alpha 特性引入，作为 CronJob 控制器的替代实现。如果选择 CronJob 控制器的 v2 版本，请在 {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}} 中设置以下[特性门控](/docs/reference/command-line-tools-reference/feature-gates/) 标志。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:173]:
```suggestion
自 Kubernetes 1.20 开始以 alpha 特性引入，作为 CronJob 控制器的替代实现。
如果选择 CronJob 控制器的 v2 版本，请在{{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
中设置以下[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)标志。
```

Comment (mysunshine92) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md]: ok, thanks, I have modified this.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -157,6 +157,28 @@ the Job in turn is responsible for the management of the Pods it represents.
...
+--feature-gates="CronJobControllerV2=true"
+```
+
+
+-->
+## 新控制器
+
+自 Kubernetes 1.20 开始以 alpha 特性引入，作为 CronJob 控制器的替代实现。
+如果选择 CronJob 控制器的 v2 版本，请在 {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+中设置以下[特性门控](/docs/reference/command-line-tools-reference/feature-gates/) 标志。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:175]:
```suggestion
中设置以下[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/) 标志。
```

please use localized version.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 3]
```diff
@@ -157,6 +157,28 @@ the Job in turn is responsible for the management of the Pods it represents.
...
+<!--
+## New controller
+
+There's an alternative implementation of the CronJob controller, available as an alpha feature since Kubernetes 1.20. To select version 2 of the CronJob controller, pass the following [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
+
+```
+--feature-gates="CronJobControllerV2=true"
+```
+
+
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:169]: These empty lines are to be removed.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 4]
```diff
@@ -157,6 +157,28 @@ the Job in turn is responsible for the management of the Pods it represents.
...
+## 新控制器
+
+自 Kubernetes 1.20 开始以 alpha 特性引入，作为 CronJob 控制器的替代实现。
+如果选择 CronJob 控制器的 v2 版本，请在 {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+中设置以下[特性门控](/docs/reference/command-line-tools-reference/feature-gates/) 标志。
+```
+--feature-gates="CronJobControllerV2=true"
+```
+
+
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:180]: These empty lines are to be removed.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md (hunk 2 above)
Comment (mysunshine92) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md]: ok, thanks, I have modified those.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 5]
```diff
@@ -157,6 +157,25 @@ the Job in turn is responsible for the management of the Pods it represents.
...
+There's an alternative implementation of the CronJob controller, available as an alpha feature since Kubernetes 1.20. To select version 2 of the CronJob controller, pass the following [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
+
+```
+--feature-gates="CronJobControllerV2=true"
+```
+-->
+## 新控制器
+
+自 Kubernetes 1.20 开始以 alpha 特性引入，作为 CronJob 控制器的替代实现。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:171]: 这句译文读起来不好，不是中文意义上的完整句子，缺主语。

Comment (mysunshine92) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md]: cc @tengqm

**Pull request merged**